### PR TITLE
fix default value of nodefs.inodesFree on linux

### PR DESCRIPTION
**Pull request by Cweiping**
Reference PR: https://github.com/kubernetes/website/pull/25934

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/out-of-resource.md
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/out-of-resource.md:122]: this sentence doesn't flow

File: content/en/docs/tasks/administer-cluster/out-of-resource.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/out-of-resource.md:144]:
```suggestion
| `PIDPressure`     | `pid.available`                                                                       | Available processes identifiers on the (Linux) node has fallen below an eviction threshold                                                        |
```

**Review by sftim — COMMENTED** [review]
/hold
@Cweiping there's a suggestion here. If you want to merge this, /unhold (and then I can open a PR to tweak the wording). You can also revise the text and get another LGTM. Either is fine.

/approve

**Review by ydFu — APPROVED** [review]

File: content/en/docs/tasks/administer-cluster/out-of-resource.md
Comment (Cweiping) [reply content/en/docs/tasks/administer-cluster/out-of-resource.md]: Done. Waiting for your review again.

File: content/en/docs/tasks/administer-cluster/out-of-resource.md
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/out-of-resource.md:144]: "process identifiers" reads pretty strange to me. Maybe "PIDs" or "process IDs" is a better option.

Comment (celestehorgan) [reply content/en/docs/tasks/administer-cluster/out-of-resource.md:144]: I don't think this is a blocking issue for merging this. "Process identifiers" just spells out what PIDs is short for.

Comment (tengqm) [reply content/en/docs/tasks/administer-cluster/out-of-resource.md:144]: @celestehorgan Yes, it is not a blocker. However, I was arguing that when we are using established terms we should follow the tradition. In all operating system text books, academic papers or even tech overviews, we don't see "process identifiers". Similarly we don't say "user identifiers" or "group identifiers", we say "UID" or "user ID" instead. In this context, "PID" or "process ID" is a special integer. "Process identifier", on the other hand, sounds like a generalization of the term. It can be other things for identifying a process.

**Comment by Cweiping** [comment]
hold cancel

File: content/en/docs/tasks/administer-cluster/out-of-resource.md
Comment (sftim) [reply content/en/docs/tasks/administer-cluster/out-of-resource.md:144]: We should write PID on this line, I think. It's OK to expand it though because PID can also mean proportional-integral-derivative, and Kubernetes uses _both_ meanings in different contexts.

**Review by sftim — COMMENTED** [review]
/lgtm

Would be nice to tidy up the PID vs. process ID detail.

**Pull request merged**
